BraveShared/Analytics/UserReferralProgram.swift

```diff
@@ -180,6 +187,19 @@ public class UserReferralProgram {
         return nil
     }
     
+    /// Passing string, attempts to derive a ref code from it.
+    /// Uses very strict matching.
+    /// Returns the sanitized code, or nil if no code was found
+    public class func sanitize(input: String?) -> String? {
+        guard var input = input, input.hasPrefix(self.clipboardPrefix) else { return nil }
+        
+        input.removeFirst(self.clipboardPrefix.count + 1)
```

Comment:
Understand this probably set in a spec somewhere, but maybe a comment about why `+ 1`?

Reply:
Nah, lol, yeah, will do!